nilearn/datasets/func.py

```diff
@@ -441,7 +441,7 @@ def fetch_adhd(n_subjects=None, data_dir=None, url=None, resume=True,
     nitrc_ids = range(7782, 7822)
     max_subjects = len(ids)
     if n_subjects is None:
-        n_subjects = max_subjects
+        n_subjects = 30
```

Comment:
Well, the changes related to pep8 are more important than the ones related to the number of subjects. I propose to separate them in 2 distinct commits.